samcli/commands/build/core/options.py

```diff
@@ -71,5 +74,6 @@
             ),
         ],
     },
+    "Hook Options": {"option_names": {opt: {"rank": idx} for idx, opt in enumerate(HOOK_OPTIONS)}},
```

Comment:
make it `Terraform Hook Options`, so in future if we decided to add more hooks, or we support 3rd party hooks we can differentiate between each hook options.

Reply:
apply this comment to other commands as well.